api/src/middlewares/services/CaseService/consularLetter/ConsularLetterCaseService.ts

```diff
@@ -0,0 +1,139 @@
+import { FormField } from "../../../../types/FormField";
+import consularLetterSubmissionTemplate from "./consularLetterSubmissionTemplate";
+import config from "config";
+import * as handlebars from "handlebars";
+import { isFieldType } from "../../../../utils";
+import { ConsularLetterProcessQueueData, PaymentData } from "../types";
+import { PaymentViewModel } from "../utils/PaymentViewModel";
+import { CaseService } from "../types";
+import { reorderSectionsWithNewName } from "../utils/reorderSectionsWithNewName";
+import { order, remap } from "./mappings";
+import { createRemapper } from "../utils/createRemapper";
+import { ConsularLetterProcessQueueDataInput } from "../types";
+import { getPostEmailAddress } from "../../utils/getPostEmailAddress";
+import { QueueService } from "../../QueueService";
+import logger, { Logger } from "pino";
+
+export class ConsularLetterCaseService implements CaseService {
+  logger: Logger;
+  queueService: QueueService;
+  templates: {
+    SES: HandlebarsTemplateDelegate;
+    Notify: Record<"postAlert", string>;
+  };
+
+  constructor({ queueService }) {
+    this.queueService = queueService;
+    this.templates = {
+      SES: ConsularLetterCaseService.createTemplate(consularLetterSubmissionTemplate),
+      Notify: {
+        postAlert: config.get<string>("Notify.Template.consularLetterPostNotification"),
+      },
+    };
+    this.logger = logger().child({ service: "SES" });
+  }
+
+  buildProcessQueueData(input: ConsularLetterProcessQueueDataInput): ConsularLetterProcessQueueData {
+    const { fields, reference, metadata, type } = input;
+    return {
+      fields,
+      metadata: {
+        reference,
+        payment: metadata.pay,
+        type,
+      },
+    };
+  }
+
+  /**
+   * This will add all the parameters needed to process the email to the queue. The NOTIFY_PROCESS queue will pick up
+   * this message and make a post request to notarial-api/forms/emails/staff
+   */
+  async sendToProcessQueue(data: ConsularLetterProcessQueueData) {
+    return await this.queueService.sendToQueue("SES_PROCESS", data);
+  }
+
+  async sendEmail(data: ConsularLetterProcessQueueData) {
+    const jobData = this.buildJobData(data);
+    return await this.queueService.sendToQueue("SES_SEND", jobData);
+  }
+
+  getEmailBody(data: { fields: FormField[]; payment?: PaymentData; reference: string }) {
+    const { fields, payment, reference } = data;
+
+    const remapFields = createRemapper(remap);
+    const remapped = remapFields(fields);
+
+    const reorderer = reorderSectionsWithNewName(order);
+    const reordered = reorderer(remapped);
+
+    const post = "the British Embassy Bangkok";
+    const applicantIsNok = remapped.information.applicantIsNextOfKin;
+    return this.templates.SES({
+      post,
+      applicantIsNok,
+      reference,
+      payment,
```

Comment:
we can remove payment from these templates since nothing should be coming through. I don't _think_ payment metadata will end up getting generated if there are no fees configured in the form either?

let's remove from the tests as well